django_netjsonconfig/base/template.py

```diff
@@ -100,5 +100,12 @@ def clean(self, *args, **kwargs):
         if self.type == 'vpn' and not self.config:
             self.config = self.vpn.auto_client(auto_cert=self.auto_cert)
 
+    def get_context(self):
```

Comment:
This, function will probably need changes, (I am not sure what this function have to output)

Reply:
right, let's take advantage of this situation to do some cleanup.

This module has a setting, `NETJSONCONFIG_CONTEXT` which contents are added to the `Config` context (see the line containing `c.update(app_settings.CONTEXT)`), but not in the result returned by `Vpn.get_context()`, nor this new method.

So the best thing and most consistent thing to do, is to add a new `get_context()` method on `django_netjsonconfig.base.base.BaseConfig`, which just does something like:

```python
def get_context(self):
    return app_settings.CONTEXT
```

Then all the other `get_context` methods on the classes inheriting from `BaseConfig` must do something like:

```python
def get_context(self):
    c = super(ModelName, self).get_context()
    # add more data to c ...
    return c
```